src/e2e/fixtures.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-empty-pattern */
+/* eslint-disable @typescript-eslint/no-empty-object-type */
+/* eslint-disable react-hooks/rules-of-hooks */
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
```

Comment:
😠😠😠

Reply:
jak już jakieś uważasz za bez sensu to wyłączaj je dla pojedynczej linijki a nie dla całego pliku